src/common/io.ts

```diff
@@ -124,6 +125,37 @@ export class IOReadable implements AoIORead {
       tags: [{ name: 'Action', value: 'Info' }],
     });
   }
+
+  async getEpochSettings(params?: EpochInput): Promise<AoEpochSettings> {
+    const allTags = [
+      { name: 'Action', value: 'Epoch-Settings' },
+      {
+        name: 'Timestamp',
```

Comment:
Have you tested this live? "Timestamp" is an overlapping tag I think.

Reply:
Timestamp is not overlapping on reads